frontend/src/app/config/local-h2ms-config.ts

```diff
@@ -9,4 +9,5 @@ export const CONFIG: Config = new Config('H2MS',
     '/assets/images/banners/h2ms-banner.png',
     '/assets/images/logos/h2ms-logo.png',
     'http://localhost',
-    8080);
+    8080,
+    [{id: null, displayName: null}]);
```

Comment:
Should be the same as the live `h2ms-config.ts` file.

Ex.
`[{id: '3', displayName: 'Washed?'}]);`